content/browser/webui/content_web_ui_controller_factory.cc

```diff
@@ -14,6 +14,9 @@
 #include "content/public/browser/web_contents.h"
 #include "content/public/browser/web_ui.h"
 #include "content/public/common/url_constants.h"
+#if defined(OS_TIZEN)
+#include "xwalk/runtime/browser/ui/webui/file_picker/file_picker_ui.h"
```

Comment:
Is okay to add WebUI files to chromium-crosswalk?
I just tried to separate the change from chromium code.